addons/website_event/models/event_tag.py

```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+from odoo import api, fields, models
+
+
+class EventTagCategory(models.Model):
+    _name = 'event.tag'
+    _inherit = ['event.tag', 'website.published.mixin']
+    event_tag_website_id = fields.Many2one(string='event_tag_website_id', store=True, related='category_id.website_id') 
```

Comment:
Maybe even readonly=1, for good measure. Doesn't seem to make sense to have a tag for specific websites if it's not in a category.